src/overmap.cpp

```diff
@@ -3951,11 +3981,33 @@ void overmap::load_oter_id_migration( const JsonObject &jo )
     }
 }
 
+void overmap::load_oter_id_camp_migration( const JsonObject &jo )
+{
+    std::string name;
+    oter_type_str_id oter;
+    faction_id owner;
+    JsonObject jsobj = jo.get_object( "camp_migrations" );
```

Comment:
I don't really get why you've nested this here, camp_migrations implies you can put multiple in one which you can't, like what's the benefit to
```
  {
    "type": "camp_migration",
    "camp_migrations": { "name": "Refugee Center", "overmap_terrain": "evac_center_13", "faction": "free_merchants" }
  },
```
over
```
  {
    "type": "camp_migration",
    "name": "Refugee Center",
    "overmap_terrain": "evac_center_13",
    "faction": "free_merchants"
  },
```

Reply:
I'm a hack and I copied from elsewhere in the codebase. "Good enough", I surely thought